### ISSUE TITLE
Reporting extended error information when loading a NES file without header

### ISSUE THREAD

**Issue by bleuge** [punesemu/puNES#39]
I've just got a Error loading rom, nothing more. 
Obviously it was a headerless rom (nointro sets). Maybe you could consider reporting this best, for example "Error loading rom, it seems it doesn't have a valid header"

Or something like that. Thanks!

**Comment by jcdenton2k** [comment]
WAIT WAIT WAIT, WAIT...wait a damn minute!
So apparently NES games need headers now?  Since when?
It isn't the job of puNES to diagnose whether your (legally acquired) ROMs have headers or not.

It is bad enough in other systems you get interlaced or deinterlaced ROMs and headered or headerless ROMs.  Really bad, such a damn mess.  The general courtesy is to have a deinterlaced headerless ROM as your standard 'good dump'.

**Comment by bleuge** [comment]
No, nes game don't need headers, if emu suppor headerless. No the job of Punes is no diagnosing anything, but emulate a NES.  My roms are perfect clean dumps, i have 2 complete sets, headed and headerless. Legally acquired? I only use freeware, PD and free released NES roms, i don't know nothing about ilegal nothing. xD

Thanks for your tips.
